unittests/dnnl/TestJsonParser.cpp

```diff
@@ -1,12 +1,22 @@
+//===-- TestJsonParser.cpp - DESC -------------------------------*- C++ -*-===//
```

Comment:
I suggest to add a description to replace "DESC" in the heading. Or maybe simply removing "DESC" is allowed in CI-checking? @WangJialei-A

Reply:
Done.